google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/git/UploadSourceAction.java

```diff
@@ -468,7 +468,8 @@ public void run() {
         return false;
       }
 
-      Collection<VirtualFile> files2add = Collections.intersection(untrackedFiles, files2commit);
+      Collection<VirtualFile> files2add =
```

Comment:
do we know that the behavior if this other utility method is the same as the one it replaces? 

I'm also a bit concerned we're using a library provided through the IDEA platform classpath that they may not have intended to be used by clients.  

will chat with you about this offline.